### PR TITLE
Cactus data highlight

### PR DESCRIPTION
**Pull request by LianeHughes**

### CODE REVIEW

File: content/highlights/cactus.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    NAISS,
+    Nextflow,
+    Docker,
+    Singularity,
+    Genomics,
+    Bioinformatics,
+    mRNA-Seq,
+    ATAC-Seq,
+  ]
+authors: [Katarina Öjefors Stark &  Jérôme Salignon]
```
Comment (senthil10) [inline content/highlights/cactus.md:24]: Just to clarify, giving it this way (i.e. with '&') Hugo will recognize it as one name. Unless it is not what we are going for it should be ["Name 1", "Name 2"] :)

**Review by senthil10 — COMMENTED** [review]
Looks good, only one small comment/clarification.

File: content/highlights/cactus.md (hunk 1 above)
Comment (LianeHughes) [inline content/highlights/cactus.md:24]:
```suggestion
authors: [Katarina Öjefors Stark, Jérôme Salignon]
```

Comment (LianeHughes) [reply content/highlights/cactus.md]: change made, thanks

**Pull request merged**